src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
+                    <i class="chi-icon icon-x"></i>
+                  </div>
+                </button>
+              </div>
+              <div class="chi-drawer__content -px--2">
+                <ul class="-no-style">
+                  <li class="-text--md">Download data</li>
+                  <div class="chi-divider"></div>
+                  <li class="-text--md">Refresh results</li>
+                  <div class="chi-divider"></div>
+                </ul>
+              </div>
             </div>
           </div>
         </div>
-      </div>
 
-      <!-- Javascript -->
-      <script>
-        chi.drawer(document.getElementById('drawer-trigger-br1'));
-        chi.drawer(document.getElementById('drawer-trigger-br2'));
-        chi.tooltip(document.querySelectorAll('[data-tooltip]'));
-      </script>
+        <!-- Javascript -->
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-brp1'));
+          chi.drawer(document.getElementById('drawer-trigger-brp2'));
+          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
+        </script>
+
+  // Drawers
+  .chi-backdrop.-closed
+    .chi-backdrop__wrapper
+      div#drawer-1.chi-drawer.-left.-portal.-menu.-position--absolute
+        .chi-drawer__header
+          span.chi-drawer__title Filters
+          button(class='chi-button -icon -close',  aria-label='Close')
+            .chi-button__content
+              i.chi-icon.icon-x
+        .chi-drawer__content.-px--2.-py--3
+          .chi-form__item.-mb--2
+            label.chi-label(for="example-ba11") Label
+            select(class="chi-select", id="example-ba11")
+              option(value="") Select
+              option Option 1
+              option Option 2
+              option Option 3
+          .chi-form__item.-mb--2
+            label.chi-label(for="example-ba12") Label
+            select(class="chi-select", id="example-ba12")
+              option(value="") Select
+              option Option 1
+              option Option 2
+              option Option 3
+          .chi-form__item.-mb--2
+            label.chi-label(for="example-ba13") Label
+            select(class="chi-select", id="example-ba13")
+              option(value="") Select
+              option Option 1
+              option Option 2
+              option Option 3
+          .chi-form__item.-mb--2
+            label.chi-label(for="example-ba14") Label
+            select(class="chi-select", id="example-ba14")
+              option(value="") Select
+              option Option 1
+              option Option 2
+              option Option 3
+        .-d--flex.-justify-content--center.-pb--2
+          button.chi-button.-primary.-lg.-uppercase.-px--4 Apply
+          button.chi-button.-primary.-outline.-lg.-bg--white.-uppercase.-px--4.-ml--2 Cancel
+  .chi-backdrop.-closed
+    .chi-backdrop__wrapper
+      div#drawer-2.chi-drawer.-right.-portal.-menu.-position--absolute
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.